### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by kakkoyun**
This PR adds better issue templates which are generated by Github.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
```
Comment (spier) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Is this empty line here on purpose?
(Same question for the other issue template)

Comment (kakkoyun) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yes, they are. I don't know the exact reason behind it but it's how GitHub generates them.

Comment (spier) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Got it.

**Comment by spier** [comment]
Nice. Works as expected:

https://github.com/meltwater/drone-cache/issues/new/choose

**Comment by kakkoyun** [comment]
@spier Great 🎉

**Pull request merged**